CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 **Changed**:
 
 - **decidim-core**: Only require default locale translations for i18n fields [\#2204](https://github.com/decidim/decidim/pull/2204)
+- **decidim-core**: Changed `ActiveJob` queue name from `decidim_events` to `events` for consistency reasons [\#2229](https://github.com/decidim/decidim/pull/2229).
```

Comment:
Should we add a warning to tell devs to re-enable the sidekiq queue?

Reply:
We don't support `sidekiq` out of the box anyways, so don't think that's needed. We should add documentation on the existing queues at some point though, so you can use whatever backend you need.